src/ui/components.tsx

```diff
@@ -4,16 +4,16 @@ import {
   AsyncSelect,
   Button,
   Checkbox,
-  Select as GrafanaSelect,
   HorizontalGroup,
   InlineField,
   InlineFieldRow,
   Input,
   Label,
+  Select as GrafanaSelect,
```

Comment:
looks like one tool sorts by the first character, the other by the character it imports from? if this causes __no__ prettier error in ci, we have to adapt the ci to make this stable.

Reply:
Seems to be an error in the ci. See [here](https://github.com/tribe29/grafana-checkmk-datasource/actions/runs/3909674284/jobs/6681008671)

Reply:
can you explain?

what exactly i the ci error? that the current main branch is green?
how can we fix this? can you create another pr for fixing this?

Reply:
Oh, what I meant was that this is caught by prettier both in the CI and locally in my tests. The linked action was a test commit where I tried and succeeded to provoke that error in particular.